README.md

```diff
@@ -46,11 +46,15 @@ spec:
       pullingInterval: "PT5S"
       validateContent: true
       validateOnlyExtraData: false
+      useTransport: true
   pins:
-    - name: read_csv_out
-      connection-type: mq
-      attributes: ['raw', 'publish', 'store']
-  extended-settings:
+    mq:
+      publishers:
+        - name: to_mstore
+          attributes:
+            - publish
+            - transport-group
+  extendedSettings:
```

Comment:
While we are here changing the readme how about changing the JAVA_TOOL_OPTIONS to something more realistic?

Reply:
Unfortunately I have run read in pico only and doesn't have all parameters for calculation more realistic parameter